### PR TITLE
Fix Tests

### PR DESCRIPTION
**Pull request by kilimanjaro**
Two tests were failing, this makes them pass. I worth noting that one of these (`test-log-backtrace`) was originally checking for the presence of a function name in the error output stream, but this seems to be missing. That might be an actual bug, in which case my quick and easy change should turn into a real bugfix.

### CODE REVIEW

File: src-tests/test-rpc.lisp [hunk 1]
```diff
@@ -289,8 +293,9 @@
            (rpcq:with-rpc-client (client addr)
              (signals rpcq::rpc-error
                (rpcq:rpc-call client "test-error-method"))
-             (is (search "Unhandled error in host program" (get-output-stream-string log-stream)))
-             (is (search "OOF-FIND-ME-ON-THE-STACK" (get-output-stream-string error-stream))))
```
Comment (kilimanjaro) [inline src-tests/test-rpc.lisp]: This is the one of interest

**Comment by ecpeterson** [comment]
I feel OK about dropping the originating function from name from the (already severely limited) trace in the error message.

**Comment by kilimanjaro** [comment]
Ironically, tests fail on gitlab...

**Comment by kilimanjaro** [comment]
Ok, after clearing my Lisp cache and updating quicklisp (`(ql:update-dist "quicklisp")`) the original tests now pass locally for me. It seems that I was originally using `cl-messagepack-20171130`, which is now updated to `cl-messagepack-20190307`. My fault for not having updated earlier!

**Pull request closed without merging**